app/styles/_story.scss

```diff
@@ -259,9 +275,20 @@ article > section a:hover {
 
 .full-bleed__body {
 
+  > .django-content {
+    font-family: 'Lato', sans-serif;
+  }
+
+  > .django-content h1, h2, h3, h4, h5, h6 {
```

Comment:
Since you’re already scoping to within the full bleed body, you don’t need > .django-content here